tox.ini

```diff
@@ -0,0 +1,9 @@
+[tox]
+envlist = py27, py35
+
+[testenv]
+commands = python {toxinidir}/imagersite/manage.py test {toxinidir}/imagersite/
```

Comment:
Goodness, this is beautiful - I'm unreasonably happy that you've hooked together tox and Django's tests.